db/install.sh

```diff
@@ -3,11 +3,14 @@
 
 POSTGRES_ADMIN=postgres
 DBUSER=postgres
-DBNAME=postgres_magpie
+DBNAME=postgres_db
 TABLENAME=magpie_cache
 
 sudo -u $POSTGRES_ADMIN createdb -O $DBUSER $DBNAME
 hostName=$(sudo netstat -plunt |grep postgres)
```

Comment:
I get an error from this line: "netstat: lunt: unknown or uninstrumented protocol"